sources/HUBViewModelDiff.h

```diff
@@ -70,8 +70,12 @@ extern HUBViewModelDiff *HUBDiffMyersAlgorithm(id<HUBViewModel>, id<HUBViewModel
 /// The index paths of any body components that were modified in the new view model. 
 @property (nonatomic, strong, readonly) NSArray<NSIndexPath *> *reloadedBodyComponentIndexPaths;
 
-/// A convenience property that returns YES if there are any inserts, deletes or reloads in this diff.
-@property (nonatomic, readonly) BOOL hasChanges;
```

Comment:
I don't think we need to change this API. We're only interested in whether there are changes - we don't really care where they come from.